README.md

```diff
@@ -119,56 +121,78 @@ command. Below is an example object using all of the available properties.
 
 ```JavaScript
 {
-  minTimes: 2,
-  recording: {
-    matchingIgnores: ['port'],
-    rewriteOrigin: 'https://not-example.com',
-    allowAllStatusCodes: true
-  }
+  allowAllStatusCodes: true,
+  toBeCalledAtLeast: 1,
+  matching: {
+    ignores: {
+      attributes: ['port'],
+      bodyProperties: ['timestamp'],
+      searchParams: ['date']
+    }
+  },
+  rewriteOrigin: 'https://not-example.com'
 }
 ```
 
-###### **playbackOptions.minTimes (number)**
+More detailed examples of how to use these properties can be found in the
+["Requests and Responses"][5] section below.
+
+###### **playbackOptions.allowAllStatusCodes (boolean)**
+
+By default, the command will only record responses that have a `2xx` status
+code. By setting this to `true`, all responses will be recorded.
+
+Note that trying to record `3xx` responses will lead to some strange behavior
+and is area where more work is needed in the plugin.
+
+*Default:* `false`
+
+###### **playbackOptions.toBeCalledAtLeast (number)**
 
-The minimum number of times the system under test must trigger a request that
-matches the `url` or `routeMatcher`. See ["All Requests Complete" Assertions][6]
-for more details.
+The minimum number of times the system under test must trigger a network request
+that matches the `url` or `routeMatcher`. See ["All Requests Complete"
+Assertions][6] for more details.
 
 *Default:* `1` - At least 1 request must have been matched.
 
-###### **playbackOptions.recording (object)**
+###### **playbackOptions.matching (object)**
 
-This object modifies the behavior of the command when it is recording a network
-response to a request.
+This object modifies how the command tries to match recorded responses to a
+network request.
 
 *Default:* `undefined`
 
-###### **playbackOptions.recording.matchingIgnores (string[])**
+###### **playbackOptions.matching.ignores (string[] | object)**
 
-An array consisting of one or more of the following values:
+This property supports two different value types:
 
-* `protocol`
-* `hostname`
-* `port`
-* `pathname`
-* `search`
-* `method`
-* `body`
+* **`string[]`:** An array of network request attributes to ignore. See the
```

Comment:
why not make the network request attributes to ignore another propery on the object intead of letting the property support two different value types? is that so anyone using it already doesn’t have to change it?

Reply:
We do support that structure, for cases where the dev wants to ignore attributes of the request as well as body properties or search params. However, I think the body properties and search params cases will be less frequent than just ignoring the attributes, especially for our own uses cases.

For example, for us, we almost always have to ignore the `hostname` and the `port`, due to testing on `localhost:<some-port>` locally and it being `cypress-target:9999` in the CI job. In those cases, I thought it would be nice to support a flatter structure.